pkg/logql/rangemapper.go

```diff
@@ -30,10 +30,16 @@ var splittableRangeVectorOp = map[string]struct{}{
 	syntax.OpRangeTypeMin:       {},
 }
 
+// RangeMapper is used to rewrite LogQL sample expressions into multiple
+// downstream sample expressions with a smaller time range that can be executed
+// using the downstream engine.
+// A rewrite is performed using the following rules:
```

Comment:
> A rewrite is performed using the following rules:

but rules are missing?

Reply:
nit: A simple example would also be very helpful IMO :) (also for `Parse` and `Map`)

Reply:
🤦‍♂️ I wanted to update the comment after writing the comments for `Map()` and `Parse()` and forgot about it.